rest-kraken/src/main/java/com/github/twitch4j/kraken/TwitchKrakenBuilder.java

```diff
@@ -110,18 +122,24 @@ public TwitchKraken build() {
             proxyConfig.apply(clientBuilder);
 
         // Build
-        TwitchKraken client = HystrixFeign.builder()
+        return HystrixFeign.builder()
             .client(new OkHttpClient(clientBuilder.build()))
             .encoder(new JacksonEncoder(mapper))
             .decoder(new JacksonDecoder(mapper))
             .logger(new Slf4jLogger())
             .logLevel(logLevel)
             .errorDecoder(new TwitchKrakenErrorDecoder(new JacksonDecoder()))
             .requestInterceptor(new TwitchClientIdInterceptor(this.clientId, this.userAgent))
+            .requestInterceptor(t -> {
+                t.header("Accept", "application/vnd.twitchtv.v5+json");
+
+                // set content-type if not specified in the TwitchKraken interface for POST, PATCH, PUT requests
+                if (!t.headers().containsKey("Content-Type") && !t.headers().containsKey("content-type") && methodsWithBody.contains(t.method())) {
+                    t.header("Content-Type", "application/json");
```

Comment:
Might be nice to define `JsonContentTypeInterceptor` (or whatever name) in the common module so it can be used in helix too